erpnext/erpnext_integrations/doctype/payment_plan/payment_plan.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+from frappe.model.document import Document
+import frappe
+from frappe import _
+from frappe.integrations.utils import create_request_log
+import stripe
+
+class PaymentPlan(Document):
+	pass
+
+
+def create_stripe_subscription(gateway_controller, data):
```

Comment:
Under erpnext_integration, create a new file stripe.py to manage erpnext sides actions.